worker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openmicroscopy/omero-server:5.6.3
+
+COPY ice.config worker.cfg /opt/omero/server/OMERO.server/etc/templates/
+
+USER root
+# Replace the default startup scripts
+RUN rm /startup/60-database.sh
```

Comment:
Could see making this easier/more explicit in the future, perhaps with an environment variable `OMERO_WORKER=1` (or test that worker_name is set??)

Reply:
We can probably come up with nicer ways to do this, but my aim in this PR was to minimise changes to the upstream Docker images.

Reply:
Understood.